sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -876,6 +877,22 @@ def reset_module(self, reset = False):
         else:
             return True
 
+    def reset(self):
+        """
+        Reset SFP and return all user module settings to their default srate.
+
+        Returns:
+            A boolean, True if successful, False if not
+        """
+        mask = (1 << 3)
+        byte = self.xcvr_eeprom.read(consts.MODULE_LEVEL_CONTROL)
```

Comment:
read call can also return None.
can you add a condition to check for None here?.

Reply:
Done